conf/application-context.xml

```
@@ -128,15 +136,10 @@
                     <constructor-arg index="2" value="#{bridgeConfig.getProperty('stormpath.neurod.directory.href')}"/>
                     <constructor-arg index="3">
                         <list>
-                            <!-- IE VirtualBox Machine, local development -->
-                            <value>10.0.2.2</value>
+                            <value>10.0.2.2</value> <!-- IE VirtualBox Machine, local development -->
                             <value>localhost</value>
                             <value>bridge-develop.herokuapp.com</value>
-                            <!-- In production however, we'd expect to have DNS
-                                entries like neurod.bridgesage.org, that we 
-                                would use instead.
-                             -->
-                            <value>bridge-prod.herokuapp.com</value>
```

You need to throw an error if the HOST header is not set correctly to detect this.